src/components/profile/profile.component.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { acronymize } from '../../utils/ether/ether';
+import tagComponent from '../../utils/helpers/tags/tags';
+import {
+  ProfileStyle,
+  ProfileNameStyle,
+  ProfileDetailsStyle,
+  ProfileAvatarStyle
+} from './profile.style';
+
+class Profile extends React.Component {
+  /** Returns the classes for the component. */
+  get classes() {
+    return classNames(
+      this.props.className
+    );
+  }
+
+  /** Returns the initials for the name. */
+  get initials() {
+    if (this.props.initials) { return this.props.initials; }
+    return acronymize(this.props.name);
+  }
+
+  /** Returns the avatar portion of the profile. */
+  get avatar() {
+    return (
+      <ProfileAvatarStyle
+        initials={ this.initials }
+        gravatar={ this.props.email }
+        size='medium-small'
+      />
+    );
+  }
+
+  /** Returns the text portion of the profile. */
+  get text() {
+    return (
+      <ProfileDetailsStyle>
+        <ProfileNameStyle data-element='name'>
+          { this.props.name }
+        </ProfileNameStyle><br />
```

Comment:
what do you think about using the `<strong>` tag instead of styling it with a styled-component? Looks like this element should be indicated as more important.

Reply:
The name has different text color in the DLS